libgalois/include/katana/PropertyGraph.h

```diff
@@ -240,11 +240,24 @@ class KATANA_EXPORT PropertyGraph {
 
   // TODO(witchel): ChunkedArray is inherited from arrow::Table interface but this is
   // really a ChunkedArray of one chunk, change to arrow::Array.
-  const std::shared_ptr<arrow::ChunkedArray>& host_to_owned_global_ids() const {
-    return rdg_.host_to_owned_global_ids();
+  const std::shared_ptr<arrow::ChunkedArray>& host_to_owned_global_node_ids()
+      const {
+    return rdg_.host_to_owned_global_node_ids();
+  }
+  void set_host_to_owned_global_node_ids(
+      std::shared_ptr<arrow::ChunkedArray>&& a) {
+    rdg_.set_host_to_owned_global_node_ids(std::move(a));
+  }
+
+  // TODO(witchel): ChunkedArray is inherited from arrow::Table interface but this is
+  // really a ChunkedArray of one chunk, change to arrow::Array.
+  const std::shared_ptr<arrow::ChunkedArray>& host_to_owned_global_edge_ids()
+      const {
```

Comment:
Well, I hate to see my TODOs multiply.  If I gave you a version of MarshalVector that returned an Array instead of a ChunkedArray, would you use it?

Reply:
Sure. I just copied this comment from the one above `host_to_owned_global_node_ids()` so I wasn't going to way in on design here.